tests/examples/aws-java-pulumiup-app/src/test/java/com/example/springboot/HelloControllerIT.java

```diff
@@ -0,0 +1,23 @@
+package com.example.springboot;
+
+import org.junit.jupiter.api.Test;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.ResponseEntity;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class HelloControllerIT {
+
+	@Autowired
+	private TestRestTemplate template;
```

Comment:
formating issue

Reply:
I copy pasted this. I'm going to add a readme with attribution